third_party/go/github_repos/BUILD

```diff
@@ -0,0 +1,7 @@
+package(default_visibility = ["PUBLIC"])
```

Comment:
why are we doing this in this way rather than as a `go_module`?

Reply:
Ah sorry this should be removed.